### ISSUE TITLE
Email ended with 'dot' before @ sign

### ISSUE THREAD

**Issue by sunupf** [mikeerickson/validatorjs#393]
Hi I meet a case when my user inputing email that ended with '.' before the @ sign will passes the validation.

here the case

```
const validator = require('validatorjs')
const rule = {email:'email'}
const data = {email:'sunupf.@gmail.com'}

const validation = new validator(data, rule)
validation.passes()
// return true
```

the problem is my backend validation (using laravel) say it false.
So, which one that actually right? is it should be pass or not?

Thanks

**Comment by mikeerickson** [comment]
Ya. I believe that this should fail. I will take a closer look to confirm.

**Comment by mikeerickson** [comment]
Fixed in pending 3.23 release (should be ready by EOD)
